site/en/docs/android/custom-tabs/guide-ui-customization/index.md

```diff
@@ -0,0 +1,99 @@
+@ColorInt int colorPrimaryDark = ContextCompat.getColor(MainActivity.this, R.color.md_theme_dark_primary);
+
+CustomTabsIntent intent = new CustomTabsIntent.Builder()
+        // set the default color scheme
+        .setDefaultColorSchemeParams(new CustomTabColorSchemeParams.Builder()
+                .setToolbarColor(colorPrimaryLight)
+                .build())
+        // set the alternative dark color scheme
+        .setColorSchemeParams(CustomTabsIntent.COLOR_SCHEME_DARK, new CustomTabColorSchemeParams.Builder()
+                .setToolbarColor(colorPrimaryDark)
+                .build())
+        .build();
+```
+
+The toolbar now has custom background and foreground colors.
+
+{% Aside 'gotchas' %}
+The Custom Tab protocol does [not support changing the color of the status bar](https://b.corp.google.com/issues/37083468).
+{% endAside %}
+
+## Configure custom enter and exit animation
+
+{% Video src="video/6hHqS5auVgWhN0cQNQztaJx5w4M2/GzQ4Bl5U3ssDtS8W6C7G.mp4", width="320", height="692", controls="true", class="screenshot screenshot--filled" %}
+Next, you can make launching (and leaving) a Custom Tab experience in your app more seamless, by defining  custom start and exit animations using <code>[setStartAnimation](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setStartAnimations(android.content.Context,int,int))</code> and <code>[setExitAnimation](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setExitAnimations(android.content.Context,int,int))</code>:
+
+```java
+CustomTabsIntent intent = new CustomTabsIntent.Builder()
+…
+.setStartAnimations(MainActivity.this, R.anim.slide_in_right, R.anim.slide_out_left)
+.setExitAnimations(MainActivity.this, android.R.anim.slide_in_left, android.R.anim.slide_out_right)
+.build();
+```
+
+## Further customizations: Title, Autohide AppBar, custom close icon, and Referrer
+
+{% Video src="video/6hHqS5auVgWhN0cQNQztaJx5w4M2/6VwbdckaWUVq9sNZAosC.mp4", width="320", height="692", class="screenshot screenshot--filled" %}
+
+There are a few more things you can do to adjust the UI of a Custom Tab to your needs.
+
+1. Hide the URL bar on scroll to give the user more space to explore web content using <code>[setUrlBarHidingEnabled](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setUrlBarHidingEnabled(boolean))(true)</code>
+2. Show the document title instead of the URL via <code>[setShowTitle](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setShowTitle(boolean))(true)</code>.
+3. Customize the close button to match the user flow in your app, for example, by showing a back arrow instead of the default <code>X</code> icon):   <code>[setCloseButtonIcon](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setCloseButtonIcon(android.graphics.Bitmap))(myCustomCloseIcon)</code>.
```

Comment:
```suggestion
* Customize the close button to match the user flow in your app, for example, by showing a back arrow instead of the default <code>X</code> icon):   <code>[setCloseButtonIcon()](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setCloseButtonIcon(android.graphics.Bitmap))(myCustomCloseIcon)</code>.
```